RecoHGCal/TICL/plugins/PatternRecognitionbyCA.h

```diff
@@ -31,6 +34,15 @@ namespace ticl {
     int missing_layers_;
     int min_clusters_per_ntuplet_;
     float max_delta_time_;
+    tensorflow::Session* eidSession_;
+    std::string eidInputName_;
+    std::string eidOutputNameEnergy_;
+    std::string eidOutputNameId_;
+    double eidMinClusterEnergy_;
```

Comment:
This ```eidMinClusterEnergy_``` is not got as parameter in ```PatternRecognitionbyCA::PatternRecognitionbyCA```, and as such it remains uninitialized: please add
By the way: why double, if it is only to be compared with float's? (see ```max_delta_time_``` above)

Reply:
✔︎ I set it to float now.